outputs.tf

```diff
@@ -26,32 +26,40 @@ output "iam_email" {
 
 output "emails" {
   description = "Map of service account emails."
-  value       = "${zipmap(var.names, google_service_account.service_accounts.*.email)}"
+  value       = zipmap(var.names, google_service_account.service_accounts.*.email)
 }
 
 output "iam_emails" {
   description = "IAM-format service account emails."
-  value       = "${zipmap(var.names, formatlist("serviceAccount:%s", google_service_account.service_accounts.*.email))}"
+  value = zipmap(
```

Comment:
Was `terraform fmt` run on this?

Reply:
Sure

Reply:
This still doesn't look like `terraform fmt` or our linting `make` task was run.

Reply:
I ran `terraform 0.12upgrade` that use `terraform fmt` as an underhood function. Then I launched `terraform fmt` separately, and nothing was changed. I used terraform-0.12.3 for that.  Which version of terraform do you use?
Also I ran terraform validate for the all directories of this project, for example:
`$~/work/gcp/forks/terraform-google-service-accounts$ terraform validate
Success! The configuration is valid.
$:~/work/gcp/forks/terraform-google-service-accounts$ pwd
/home/weannye/work/gcp/forks/terraform-google-service-accounts`
About lint tests, it looks outdated and it can't work even with shellcheck, and I think it should be a separate task like `update and standardization of tests`

Reply:
I have noticed some inconsistencies in behaviour with `terraform fmt` on Terraform 0.12 that include examples like this.